app/views/player.py

```diff
@@ -142,18 +149,18 @@ class PlayerListView(View):
     template_name = 'dashboard/player_list.html'
 
     def get(self, request):
-        players = Player.objects.filter(active=True).all()
+        participants = Participant.objects.filter(active=True).select_subclasses('player', 'spectator', 'moderator')
```

Comment:
should we sort this?

Reply:
how would we sort it? also it'd be nicer if there was sorting on the UI in the first place

Reply:
we'll just talk about adding sorting later (perhaps on the UI, since sorting in the backend is fairly unimportant).